internal/org/model/org.go

```diff
@@ -0,0 +1,52 @@
+package model
+
+import (
+	es_models "github.com/caos/zitadel/internal/eventstore/models"
+)
+
+type Org struct {
+	es_models.ObjectRoot
+
+	State  OrgState
+	Name   string
+	Domain string
+
+	Members []*OrgMember
+}
+
+type OrgState int32
+
+const (
+	ORGSTATE_ACTIVE OrgState = iota
+	ORGSTATE_INACTIVE
+)
+
+func NewOrg(id string) *Org {
+	return &Org{ObjectRoot: es_models.ObjectRoot{AggregateID: id}, State: ORGSTATE_ACTIVE}
+}
+
+func (o *Org) IsActive() bool {
+	return o.State == ORGSTATE_ACTIVE
+}
+
+func (o *Org) IsValid() bool {
+	return o.Name != "" && o.Domain != ""
+}
+
+func (o *Org) ContainsMember(userID string) bool {
+	for _, member := range o.Members {
+		if member.UserID == userID {
+			return true
+		}
+	}
+	return false
+}
+
+func (o *Org) MemberByUserID(userID string) *OrgMember {
+	for _, member := range o.Members {
+		if member.UserID == userID {
+			return member
+		}
+	}
+	return nil
+}
```

Comment:
if we want this methods every where the same, it should be getMember and return index, member

Reply:
the method isn't used anyway. I deleted it